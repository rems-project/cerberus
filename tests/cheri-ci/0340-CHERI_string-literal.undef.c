int main(void)
{
  char *str = "string";
  str[0];
  str[0] = 'S'; // undefined
}
