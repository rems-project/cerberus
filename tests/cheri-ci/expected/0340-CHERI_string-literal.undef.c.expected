Undefined {ub: "UB033_modifying_string_literal", stderr: "", loc: "<5:3--5:15>"}
